source/mainnet/conf.py

```diff
@@ -84,9 +85,15 @@
 copybutton_prompt_text = "$"
 copybutton_line_continuation_character = "\\"
 
+# Disable default url prefix "/en/latest/" on every resource.
+notfound_urls_prefix = "/concordium.github.io/"
```

Comment:
```suggestion
notfound_urls_prefix = "/"
```
Setting this should fix the image